docker/flux/Dockerfile

```diff
@@ -1,8 +1,10 @@
 ARG BASE_IMAGE=exaworks/sdk-base
 FROM ${BASE_IMAGE}
 
-ARG FLUX_CORE_VERSION=0.26.0
-ARG FLUX_SCHED_VERSION=0.16.0
+ARG FLUX_CORE_VERSION=0.28.0
+ARG FLUX_SCHED_VERSION=0.17.0
+ENV FLUX_CORE_VERSION ${FLUX_CORE_VERSION}
+ENV FLUX_SCHED_VERSION ${FLUX_SCHED_VERSION}
 
 # Install Flux-core from release tarball
 RUN V=$FLUX_CORE_VERSION \
```

Comment:
is it possible to install flux python package within virtual environment (`$VIRTUAL_ENV`)?
with the current installation it is installed here `/usr/lib64/python3.6/site-packages/`, but would be expected here ` /ve_exaworks/lib64/python3.6/site-packages/`

Reply:
Unfortunately, Flux doesn't have a way to install the main part and python parts to separate prefixes.  It either all gets installed in `/usr/` or all installed in `/ve_exaworks/`.

Reply:
(Sorry, hit comment prematurely)

I'm not sure what the right way forward is.  One option is to do nothing and leave things the way they are.  In this case, when someone runs the `flux` command, it prepend a special flux directory to the PYTHONPATH to ensure that the flux packages are importable:
```
ƒ(s=1,d=0) fluxuser@bcb80b383c1b:~$ flux env | grep PYTHONPAT
export PYTHONPATH="/usr/lib/flux/python3.6"
```

Note that `/usr/lib/flux/python3.6` != `/usr/lib64/python3.6/site-packages/`, so this action does not "pollute" the virtualenv with any system packages.  We also have a convenience command to launch the python that flux was configured with: `flux python`.

Another option is to install all of Flux into the virtualenv.  I've never done that with software before, but I'm open to the idea if it solves some consistency issues.

Another option is to make changes to Flux upstream that allows us to install the python bits in a different place from the main bits, but if you choose a virtualenv for the python bits and a system-wide directory for the main bits, weirdness will probably ensue.

Thoughts?

Reply:
I like the idea with setting a path into `PYTHONPATH` (thus not to break any installation steps) and agree with flux-path

should the following line be added to flux dockerfile (right after installation done)?
`ENV PYTHONPATH="/usr/lib64/flux/python3.6:${PYTHONPATH}"`

not sure about ending (`:${PYTHONPATH}`) since by default this env variable it empty

I did check that it is `lib64` and that able to import `flux` within python

Reply:
Sounds good to me.  I'll add that now.